html/main.js

```diff
@@ -0,0 +1,183 @@
+        "                                </div>\n" +
+        "                                <div class=\"meta\">\n" +
+        "                                    <div class=\"ui transparent fluid input meta\">\n" +
+        "                                        <input type=\"text\" placeholder=\"Desription ...\" value=\"" + description + "\">\n" +
+        "                                    </div>\n" +
+        "                                </div>\n" +
+        "                            </div>\n" +
+        "                            <div class=\"extra content\">\n" +
+        "                                <div class=\"description\">\n" +
+        "                                    <div class=\"ui transparent fluid input\">\n" +
+        "                                        <input type=\"text\" placeholder=\"Your command ...\" value=\"" + command + "\">\n" +
+        "                                        <div class=\"ui left pointing label olive\" style=\"display: none\">Keep it short\n" +
+        "                                        </div>\n" +
+        "                                    </div>\n" +
+        "                                </div>\n" +
+        "                            </div>\n" +
+        "                            <div class=\"extra content\">\n" +
+        "                                <div class=\"description\">\n" +
+        "                                    <div class=\"ui transparent fluid input\">\n" +
+        "                                        <input type=\"text\" placeholder=\"Output text ...\" value=\"" + output + "\">\n" +
+        "                                    </div>\n" +
+        "                                </div>\n" +
+        "                            </div>\n" +
+        "                            <div class=\"extra content\">\n" +
+        "                                <div class=\"ui two buttons\">\n" +
+        "                                    <div class=\"ui basic grey button disable\">Disable</div>\n" +
+        "                                    <div class=\"ui basic red button delete\">Delete</div>\n" +
+        "                                </div>\n" +
+        "                            </div>\n" +
+        "                        </div>";
+
+    let existingDiv = $("#cards-parent > div").last();
+    if (existingDiv.length) {
+        existingDiv.after(newcard);
+    } else {
+        cardsParent.append(newcard);
+    }
+
+    if (modalUi.modal("is active")) {
+        modalUi.modal("hide");
+    }
+}
+
+modalUi.modal("setting", "transition", "horizontal flip").modal({
+    closable: true,
+    onApprove() {
+        return false;
+    },
+    onShow() {
+        modalForm.form("clear");
+    }
+});
+
+function sendCommand(kind, operation, data) {
+    let obj = {
+        kind: kind,
+        operation: operation,
+        data: data
+    };
+
+    send(obj);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['send' is not defined.](https://app.codacy.com/manual/ttimt/Short_Cutteer/pullRequest?prid=4845233)